### ISSUE TITLE
Empty metadata for AAC files

### ISSUE THREAD

**Issue by hvianna** [Borewit/music-metadata-browser#126]
Hello! 

I was working on an audioMotion update and realized it's not fetching metadata from any aac/m4a files anymore. `parseBlob()` still works fine, but not `fetchFromUrl()` or `parseReadableStream()`. No issues with other formats, like mp3, flac and ogg. 

I'm not sure which version introduced the issue, since I use mostly flac. I had some old test code working with music-metadata-browser **v1.4.0**, and it stops working if I update to v1.5.0. The problem persists up to v1.9.1. 

Below is the output with music-metadata-browser@1.4.0:

![metadata_ok](https://user-images.githubusercontent.com/1033735/70752095-a7325f80-1d10-11ea-8adc-5ea092d9951c.png)

After updating to 1.5.0, the metadata is returned empty for m4a files:

![empty_metadata](https://user-images.githubusercontent.com/1033735/70752179-d8129480-1d10-11ea-8a0f-a259c3765de8.png)

Here with debug info.. the parser seems to be working fine, but the collector returns an empty format/codec..

![metadata_debug](https://user-images.githubusercontent.com/1033735/70752218-e5c81a00-1d10-11ea-88a7-7f7ce3d4184a.png)

My test code:

```js
const mm = require('music-metadata-browser');

let files = [ 'test.m4a', 'test.mp3', 'test.ogg', 'test.flac', 'http://a1rj.streams.com.br:7801/stream' ];

files.forEach( uri => {

	mm.fetchFromUrl( uri, { duration: false, skipCovers: true } ).then( metadata => {
		console.log( `${uri} via fetchFromUrl():`, metadata );
	}).catch( err => console.log( err ) );

	fetch( uri ).then( response => {
		return response.body;
	}).then( stream => {
		mm.parseReadableStream( stream, '', { duration: false, skipCovers: true } ).then( metadata => {
			console.log( `${uri} via parseReadableStream():`, metadata );
			stream.cancel();
		}).catch( err => console.log( err ) );
	});

});
```

**Comment by Borewit** [comment]
Thank you so much for your detailed analysis @hvianna, I will investigate.

**Comment by Borewit** [comment]
Should be fixed in [v1.9.2](https://github.com/Borewit/music-metadata-browser/releases/tag/v1.9.2).
